cves/kernel/CVE-2021-45868.yml

```diff
@@ -356,8 +360,8 @@ forgotten_check:
     Answer must be true or false.
     Write a note about how you came to the conclusions you did, regardless of
     what your answer was.
-  answer:
-  note:
+  answer: true
+  note: This vulnerability was caused by a lack of varrible checking.
```

Comment:
Variable Spelling, and it might be interesting to talk about the check that was added to mitigate the vulnerability.